v1.2/drop-sequence.md

```diff
@@ -28,11 +28,39 @@ table td:first-child {
 
  Parameter | Description
 -----------|------------
+`IF EXISTS` |  Drop the sequence only if it exists; if it does not exist, do not return an error.
+`sequenceName` | The name of the sequence you want to drop. Find the sequence name with `SHOW CREATE TABLE` on the table that uses the sequence.
+`CASCADE` | Drop all objects (such as [constraints](constraints.html) and tables) that depend on the sequence.<br><br>`CASCADE` does not list objects it drops, so should be used cautiously.
```

Comment:
Will CASCADE drop the table(s) that use the sequence?

Reply:
Apparently yes; that's what PG does. (Of course, I haven't implemented this yet)